spec/models/user_spec.rb

```diff
@@ -51,4 +51,27 @@
       expect(user.signed_in_before).to be false
     end
   end
+
+  describe 'appointments_count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.